recipes-domd/agl/files/meta-xt-prod-extra/recipes-graphics/wayland/weston-conf.bbappend

```diff
@@ -1,74 +0,0 @@
-FILESEXTRAPATHS_append := ":${THISDIR}/weston"
```

Comment:
was this bbappend added when you split weston recipe and now you remove it?

Reply:
as i mentioned above weston config generation was changed, so finally this recipe became unneeded.

Reply:
squashed